proto/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google LLC
```

Comment:
will these files in `proto/google/api/` need to be included in the repo now?

Reply:
grpc-gateway v2.3.0 removed the `third_party` files; [the easiest work-around is to copy them in](https://github.com/grpc-ecosystem/grpc-gateway/issues/1935#issuecomment-803572170).

Reply:
sounds good, thanks for the info!